radio/src/telemetry/frsky_sport.cpp

```diff
@@ -186,7 +186,17 @@ void sportProcessTelemetryPacket(uint8_t * packet)
           sportProcessTelemetryPacket(id, 1, instance, data >> 16);
         }
         else if (id >= RBOX_STATE_FIRST_ID && id <= RBOX_STATE_LAST_ID) {
-          uint16_t newServosState = data & 0xffff;
+          bool static isRB10 = false;
+          uint16_t newServosState;
+
+          if ((servosState & 0xff00) == 0 && (data & 0xff00) == 0xff00) {
```

Comment:
Not sure I have the latest spec, which is the bit to indicate we have discussing with an RB10?